src/coreclr/src/gc/unix/cgroup.cpp

```diff
@@ -348,6 +375,50 @@ class CGroup
         return cgroup_path;
     }
 
+    static bool GetCGroupMemoryLimit(uint64_t *val, const char *filename)
+    {
+        bool result = false;
```

Comment:
A nit - can you please remove the result definition from here and return `false` in the two following checks and define the result in the assignment at line 389 below? The pattern that's here would be useful if there was a single return point from this function, but since there are multiple return points, I would prefer to do it the way I've suggested.

The same for the GetCGroupMemoryUsage method .